tests/libtest.sh

```diff
@@ -138,6 +138,7 @@ assert_file_has_content () {
     if ! grep -q -e "$2" "$1"; then
         sed -e 's/^/# /' < "$1" >&2
         echo 1>&2 "File '$1' doesn't match regexp '$2'"
+	cat $1
```

Comment:
Were you reading the "squashed" diff?  I split this into its own commit in e39f4188f644fba77888433a4b93c0da70a33eb3